3rd_party/oxygine-framework/oxygine/res/ResAtlasGeneric.cpp

```diff
@@ -7,6 +7,7 @@
 
 #include "coreengine/gameconsole.h"
 #include "coreengine/settings.h"
+#include "coreengine/vfs.h"
```

Comment:
Please remain the class and file to VirtuellFilesystem.
Abbreviations are in most cases forbidden. They reduce readability and auto completion makes longer names no problem.

Reply:
"Vfs" is a fairly common abbreviation within codebases where this makes sense, for a class that I'd expect to be used fairly often across the codebase (once all the existing code that manually constructs search paths/etc is accounted for). Why is this abbreviation not acceptable?

Reply:
Abbreviations don't add readability and due to IDE's auto completion you don't type more words.
And the length of the line isn't really sky rocking.
The amount of usages don't play a role if a word should be abbreviated or not.
And in the coding guidelines in my work and here I want to avoid abbreviations unless the variable or function gets out of hand. This isn't the case here.